src/cf-layout/src/organisms/featured-content-module.less

```diff
@@ -0,0 +1,101 @@
+
+@fcm-bp-max: 700px;
+@fcm-bp-min: 701px;
+
+.o-featured-content-module {
+  .webfont-regular();
+
+  position: relative;
+
+  min-height: 320px;
+  margin-right: -1px;
+  margin-left: -1px;
+
+  background-color: @block__bg;
+
+  .respond-to-max( @bp-xs-max, {
+    padding-right: 15px;
```

Comment:
Should this be converted to ems? I know it wasn't in the original, but that might be an oversight.